examples/resource_extra_fields_export.py

```
@@ -0,0 +1,225 @@
+
+# Automatically adjust column width based on content length
+for col in ws.columns:
+    max_length = 0
+    column = col[0].column
+    column_letter = get_column_letter(column)
+
+    for cell in col:
+        try:
+            if cell.value:
+                max_length = max(max_length, len(str(cell.value)))
+        except:
```

Ruff (0.15.2)

[error] Do not use bare `except`

(E722)